src/main/java/de/tum/cit/aet/artemis/core/service/telemetry/TelemetrySendingService.java

```diff
@@ -0,0 +1,131 @@
+package de.tum.cit.aet.artemis.core.service.telemetry;
+
+import static de.tum.cit.aet.artemis.core.config.Constants.PROFILE_SCHEDULING;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.env.Environment;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import de.tum.cit.aet.artemis.core.service.ProfileService;
+
+@Service
+@Profile(PROFILE_SCHEDULING)
+public class TelemetrySendingService {
+
+    private static final Logger log = LoggerFactory.getLogger(TelemetrySendingService.class);
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public record TelemetryData(String version, String serverUrl, String operator, List<String> profiles, boolean isProductionInstance, String dataSource, boolean isMultiNode,
+            long buildAgentCount, String contact, String adminName) {
+    }
+
+    private final Environment env;
+
+    private final RestTemplate restTemplate;
+
+    private final ProfileService profileService;
+
+    public TelemetrySendingService(Environment env, RestTemplate restTemplate, ProfileService profileService) {
+        this.env = env;
+        this.restTemplate = restTemplate;
+        this.profileService = profileService;
+    }
+
+    @Value("${artemis.version}")
+    private String version;
+
+    @Value("${server.url}")
+    private String serverUrl;
+
+    @Value("${info.operatorName}")
+    private String operator;
+
+    @Value("${info.operatorAdminName}")
+    private String operatorAdminName;
+
+    @Value("${info.contact}")
+    private String operatorContact;
```

Comment:
is there a reason why you changed this from `contact` (as in the `info` object) to `operatorContact`?

Reply:
Sooo wouldn't it make more sense to leave it as `content` in the code too?